tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+from src.bookmarker.core.database import DatabaseRepository, SQLModel, create_engine
+
+
+@pytest.fixture()
+def db_repo() -> DatabaseRepository:
+    engine = create_engine("sqlite:///:memory:", echo=False)
```

Comment:
as per above, consideration: move engine creation also in the constructor of the db class?